Automattic/wp-calypso
# Add Product Upgrade Map

#### Changes proposed in this Pull Request

* Allow users to upgrade from Jetpack Backup from Daily to Realtime.
After the user bought the Dailyplan already.

#### Testing instructions
* Buy the daily plan. 
* Notice that you can buy the Realtime product. 
* Buy the realtime plan. 
* Notice that you can't buy the Daily Prpoduct.

* Buy the Personal Plan. 
* Notice that you can buy the realtime product still.
